client/server/pages/index.js

```diff
@@ -645,6 +651,13 @@ module.exports = function() {
 	app.use( setupLoggedInContext );
 	app.use( handleLocaleSubdomains );
 
+	if ( jetpackCloudEnvs.includes( process.env.CALYPSO_ENV ) ) {
```

Comment:
Is this always `CALYPSO_ENV`? I sometimes see `NODE_ENV`. Perhaps it would be more flexible to check the `config` env instead of directly reading the process?

Reply:
Really good catch @ChaosExAnima! Fixed in https://github.com/Automattic/wp-calypso/pull/39058/commits/6a4d6750c185c560ae92f83fb3b941aed596d581